server/src/sessionManagement.mjs

```diff
@@ -31,13 +32,16 @@ class FireboltCall {
     }
 }
 
-class Session{
+class Session {
     #sessionStart;
     #sessionEnd;
     
     constructor(){
         this.calls = [];
         this.#sessionStart = Date.now();
+        this.sessionOutput = "log";
+        this.sessionOutputPath = "./output/sessions";
```

Comment:
Still a tad concerned about the use of ".". Won't that put this wherever someone is running? Don't know if that's a feature or a bug :). I guess it's okay if we use "." here as long as we use "." everywhere we create output files/directories.

Reply:
So as far as I know (I may be wrong) it seems that to run the server you need be in the mock-firebolt/server in order to access the package.json scripts. So when this is running I would think the '.' would always be server. But I can change to an absolute path if that would be safer.